TWLight/resources/views.py

```diff
@@ -63,63 +66,27 @@ def get_context_data(self, **kwargs):
                     "are a staff member, but it is not visible to non-staff "
                     "users."))
 
-        context['total_apps'] = Application.objects.filter(
-            partner=partner).count()
-
-        context['total_apps_approved'] = Application.objects.filter(
-            partner=partner, status=Application.APPROVED).count()
-
-        context['total_apps_sent'] = Application.objects.filter(
-            partner=partner, status=Application.SENT).count()
+        all_auths_partner = Authorization.objects.filter(partner=partner)
```

Comment:
assuming you update your get_valid_authorizations in helper to return a valid queryset, you could then just chain the queryset with another filter on partner.pk here
https://docs.djangoproject.com/en/1.11/topics/db/queries/#chaining-filters